schema/artwork/index.js

```diff
@@ -309,8 +309,37 @@ const ArtworkType = new GraphQLObjectType({
       availability: {
         type: GraphQLString,
       },
+      is_on_hold: {
+        type: GraphQLString,
+        resolve: ({ availability }) => availability === 'on hold',
+      },
+      is_not_for_sale: {
+        type: GraphQLString,
+        resolve: ({ sale_message }) => sale_message === 'Not for sale',
+      },
       sale_message: {
         type: GraphQLString,
+        args: {
+          format: {
+            type: GraphQLBoolean,
+            default: false,
+            description: 'Show a formatted response instead of the one that comes from gravity',
+          },
+        },
+        resolve: (artwork, { format }) => {
```

Comment:
Right so my thought is, in places where we want this updated format, we just set `format: true` in the MP query, otherwise we can leave it alone.